pages/index.vue

```diff
@@ -23,6 +36,23 @@ export default {
     Days,
     Goals,
   },
+  data() {
+    return {
+      theme: '',
+    }
+  },
+  beforeMount() {
+    const currentTheme = getCookie('theme')
+    if (currentTheme) {
+      this.theme = currentTheme
+      this.theme === 'dark'
+        ? (document.documentElement.className = 'dark')
```

Comment:
Is there any problem with this way?
`document.documentElement.className = this.theme === 'dark' ? 'dark':'light'`

Reply:
Right should have made it simple